### DISCUSSION TITLE
Question to OSP Suite V10

### DISCUSSION THREAD

**Discussion by MSmodeler** [Open-Systems-Pharmacology/Forum#973] [Questions & Problems] [answered]
Dear all, 

I saw that you announced beta Version of OSPSuite V10. I had a look at the new features and therefore I got a few questions regarding the redesigned protein expression section. As I read the documentation a expression of an enzyme is now also available in plasma. Is this also possible for protein binding partners? This would be nice to create expression of hormone binding enzymes (e.g SHBG or TBG). 
And my second question is when Version 10 will be available as a full version and also if it is possible to open and edit a project which was created in PK Sim version 9. 

Thank you for your work and answers!

**Comment by StephanSchaller** [comment, marked as answer]
Yes, also for protein binding partners.

V10 is now close to a release (2-4 Weeks, please anyone else correct me here if this is not right) and yes, V9 projects will of course still be usable.

**Reply by MSmodeler** [reply]
Thank you @StephanSchaller for your reply. These are great news. 
This will make my modeling approach way easier than before. At the moment I had to define SHBG as a compound and export the simulation to MOBI to create binding to the relevant substances. This will now be unnecessary with V10.
